### ISSUE TITLE
Add node and npm version in readme

### ISSUE THREAD

**Issue by TheOneAboveAllTitan** [coding-blocks/boss#187]
There are lots of issue of people downloading wrong version of node and then creating new issue. Adding versions of these packages will help all developers.

**Comment by abhishek97** [comment]
Yes, it is good idea indeed. I'll suggest using `.nvmrc` in root directory and updating docs to recommend using the same.

**Comment by sakshamtaneja21** [comment]
yes a good idea

**Comment by farhan787** [comment]
@abhishek97 please review :-)

**Comment by GOYALAYUSH782** [comment]
@abhishek97 Is this issue open? I would like to to work on it.

**Comment by TheOneAboveAllTitan** [comment]
@GOYALAYUSH782 I guess this issue is open and two pull request are already open

**Comment by mritunjaysharma394** [comment]
The semantic error propped up while using the updated version of node.js. After various checks using :
https://nodejs.org/dist/ it was found that version 10.4.1 worked fine. So I #193

**Comment by TheOneAboveAllTitan** [comment]
Even the BOSS has ended. Can you please merge it now @abhishek97

**Comment by sarthakkundra** [comment]
Is this issue still open? I would like to work on it

**Comment by arjunbansal090** [comment]
I would like to work on this issue

**Comment by Akansha2202** [comment]
i want to work on this .

**Comment by heysujal** [comment]
how do i pull?

**Comment by pandafy** [comment]
Hey guys, as you can see there are already two pull requests open for this issue. Unless your work improves those PRs it will be pointless to open a new one. IMO, it is better to focus on other issues.
